__tests__/app.test.js

```diff
@@ -124,3 +123,56 @@ describe.only("get/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("post/api/articles/:article_id/comments", () => {
+  test("POST: 201 should post a comment object to corresponding article using id and respond with that object", () => {
+    const newComment = {
+      body: "This is the new comment on article 9",
+      author: "butter_bridge",
+    };
+    return request(app)
+      .post("/api/articles/9/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body[0]).hasOwnProperty("body");
+        expect(body[0]).hasOwnProperty("author");
+        expect(body[0]).hasOwnProperty("article_id");
+      });
+  });
+  test("POST:404 should respond with correct msg for invalid id", () => {
+    const newComment = {
+      body: "This is the new comment on article 9",
+      author: "butter_bridge",
+    };
+    return request(app)
+      .post("/api/articles/999/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
+  });
+  test("POST:404 should respond with correct msg for invalid id", () => {
```

Comment:
These two tests have the same description?

Also, is the id actually invalid? It looks like a valid ID just that it doesn't exist?